zdm/tests/Scattering/test_scattering_methods.py

```diff
@@ -0,0 +1,125 @@
+
+
+#import pytest
+
+#from pkg_resources import resource_filename
+import os
+#import copy
+#import pickle
+
+#from astropy.cosmology import Planck18
+
+from zdm import cosmology as cos
+from zdm import misc_functions
+from zdm import parameters
+from zdm import survey
+from zdm import pcosmic
+from zdm import iteration as it
+from zdm.craco import loading
+from zdm import io
+
+#from IPython import embed
+
+import numpy as np
+from zdm import survey
+from matplotlib import pyplot as plt
+
+def main():
```

Comment:
name this something like `def test_scatt_blah`
  Needed for CI

Reply:
And you may need to turn off the plotting..

Reply:
Oh, oops. Plots are important. Anyway, added a tweak to toggle plotting on/off in program. Dodgy but oh well.

Reply:
Btw I did a 2D plot check of before/after, and everything is fine. The "new" method here is actually CHIME's parameters, which result in FRBs which are not as wide at even 892 MHz, hence the 'new' method results in more FRBs, particularly at low DM where DM smearing is not so important. However our measurements show wider FRBs than CHIME sees, hence I revert to our old width description PLUS scattering.